problems/1.types+operators.js

```diff
@@ -2,19 +2,34 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+function isNumber(val) {
+  if (val === val) {
+    return true
+} else if (val === typeof Number) {
+  return true
+} else {
+  return false 
+}
 }
 
+
+
 /** Takes in a value and returns whether it is truthy
 * @param {any} val - The input value
 * @returns {boolean} - Is val truthy
 */
-function isTruthy() {
-
+function isTruthy(val) {
+if (val === val) {
```

Comment:
same as above. But this is checking only for truthyness. Remember the only falsy values in JS are "", 0, false, NaN, null, undefined